src/scenes/start-scene.js

```diff
@@ -142,12 +143,40 @@ export class StartScene extends Phaser.Scene {
             }
         });
 
-
+        this.randomizeButton = new TextButton(this, 50, 300, 'RANDOMIZE SEATING', {
+            onClick: () => {
+                // Make an array all the seats and randomly pick them off for each player
+                this.removeInactivePlayers() // FIXME normally for every player, there are ~2 additional inactive players. This manages to remove all of them
+                let emptySeats = []
+                this.seats.forEach(seat => {
+                    emptySeats.push(seat);
+                });
+                console.log('players: ', this.playersLabel.playerLabels) // FIXME used for debugging inactive player problem
+                this.playersLabel.playerLabels.forEach(label => {
+                    let randomIndex = Math.floor(Math.random()*emptySeats.length);
```

Comment:
Use `Phaser.Math.RND` instead

Reply:
Done! Is it better than the inbuilt random function?